source/Nevermore/RelationalStoreConfiguration.cs

```diff
@@ -23,11 +23,11 @@ public class RelationalStoreConfiguration : IRelationalStoreConfiguration
         readonly Lazy<string> connectionString;
 
 #nullable enable
-        public RelationalStoreConfiguration(string connectionString, IPrimaryKeyHandlerRegistry? customPrimaryKeyHandlerRegistry = null) : this(() => connectionString, customPrimaryKeyHandlerRegistry)
+        public RelationalStoreConfiguration(string connectionString, IPrimaryKeyHandlerRegistry? customPrimaryKeyHandlerRegistry = null, ITableNameResolver? tableNameResolver = null) : this(() => connectionString, customPrimaryKeyHandlerRegistry, tableNameResolver)
```

Comment:
since this is not a pre-req and the prop is settable, we don't gain much from including it in the ctor. I notice we've used this pattern before, I think it might have just been missed that time.